src/Finder.php

```diff
@@ -654,6 +654,92 @@ public function listFeatures($serviceName = '')
     }
 
     /**
+     * Get the path to the passed model.
+     *
+     * @param string $model
+     * @return string
+     */
+    public function findModelPath($model)
+    {
+        return $this->getSourceDirectoryName() . '/Data/' . $model . '.php';
+    }
+
+    /**
+     * Get the path to the policies directory.
+     *
+     * @return string
+     */
+    public function findPoliciesPath()
+    {
+        return $this->getSourceDirectoryName() . '/Domains/Policies';
```

Comment:
I was wondering whether `Domains` is the best place for the `Policies` to exist. Or whether it should have its own directory (i.e. `src/Policies`). Would love to hear your thoughts on that.